grype/lib.go

```diff
@@ -29,11 +29,13 @@ func FindVulnerabilities(provider vulnerability.Provider, userImageStr string, s
 		return match.Matches{}, pkg.Context{}, nil, err
 	}
 
-	return FindVulnerabilitiesForPackage(provider, context.Distro, packages...), context, packages, nil
+	matchers := matcher.NewDefaultMatchers(matcher.Config{})
+
+	return FindVulnerabilitiesForPackage(provider, context.Distro, packages, matchers), context, packages, nil
 }
 
-func FindVulnerabilitiesForPackage(provider vulnerability.Provider, d *linux.Release, packages ...pkg.Package) match.Matches {
-	return matcher.FindMatches(provider, d, packages...)
+func FindVulnerabilitiesForPackage(provider vulnerability.Provider, d *linux.Release, packages []pkg.Package, matchers []matcher.Matcher) match.Matches {
+	return matcher.FindMatches(provider, d, packages, matchers)
```

Comment:
This seems odd to me: a `matcher` and `matchers`

Reply:
FindMatches uses a `provider`, `distro`, and `matchers` to return a list of matches from `packages`

It reads ok to me but I'm open to suggestions on the refactor.